src/cmd.c

```diff
@@ -1089,6 +1090,139 @@ static void pctx(atci_param_t *param)
     request_confirmation = true;
 }
 
+static void cw(atci_param_t *param)
+{
+    uint32_t freq, timeout;
+    int32_t power;
+
+    if (!atci_param_get_uint(param, &freq)) abort(ERR_PARAM);
+    if (!atci_param_is_comma(param)) abort(ERR_PARAM);
+    if (!atci_param_get_int(param, &power)) abort(ERR_PARAM);
+    if (!atci_param_is_comma(param)) abort(ERR_PARAM);
+    if (!atci_param_get_uint(param, &timeout)) abort(ERR_PARAM);
+
+    log_debug("$CW: freq=%ld power=%ld timeout=%ld\r\n",  freq, power, timeout);
+
+    MlmeReq_t mlr = { .Type = MLME_TXCW };
+    mlr.Req.TxCw.Timeout = timeout;
+    mlr.Req.TxCw.Frequency = freq;
+    mlr.Req.TxCw.Power = power;
+
+    abort_on_error(LoRaMacMlmeRequest(&mlr));
+
+    OK_();
+}
+
+
+static void SX1276SetOpMode( uint8_t opMode )
+{
+    if( opMode == RF_OPMODE_SLEEP )
+    {
+        SX1276SetAntSwLowPower( true );
+    }
+    else
+    {
+        // Enable TCXO if operating mode different from SLEEP.
+        SX1276SetBoardTcxo( true );
+        SX1276SetAntSwLowPower( false );
+        SX1276SetAntSw( opMode );
+    }
+    SX1276Write( REG_OPMODE, ( SX1276Read( REG_OPMODE ) & RF_OPMODE_MASK ) | opMode );
+}
```

Comment:
Let's try to get rid of this duplicate function. I can add a local modification to LoRaMac-node to export this function, or we can try to switch to some of the higher-layer wrappers that invoke this function internally (please see below).

What do you think, @hubmartin ?

Reply:
I'll be happy to get rid of this. If you export `SX1276SetTx` then this might be deleted.

Reply:
Tried to add PR but something went wrong and whole content of the file went as a change instead of few lines..